colcon_sanitizer_reports/event_handlers/sanitizer_report.py

```diff
@@ -58,3 +58,6 @@ def _handle(self, event) -> None:
 
         with open('sanitizer_report.csv', 'w') as report_csv_f_out:
             report_csv_f_out.write(self._log_parser.get_csv())
+
+        with open('test_results.xml', 'w') as report_xml_f_out:
+            report_xml_f_out.write(self._log_parser.get_xml())
```

Comment:
See my comments on the logic to write csv files which partially apply here: https://github.com/colcon/colcon-sanitizer-reports/pull/4#pullrequestreview-239636349

Reply:
Created #14.